src/Algebra/Module/Definitions/Bi/Simultaneous.agda

```diff
@@ -0,0 +1,18 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties connecting left-scaling and right-scaling over the same scalars
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Relation.Binary
+
+module Algebra.Module.Definitions.Bi.Simultaneous
+  {a b ℓb} (A : Set a) {B : Set b} (_≈_ : Rel B ℓb)
```

Comment:
So I'm a bit confused about the lack of symmetry here. I would expect the implicit statues to the be the same over both set arguments in a module called `Bi`. Indeed these module parameters are the same as those in `Algebra.Module.Definitions.Left`? Is there a good explanation for this?

Reply:
This module defines a property that relates both left and right multiplication (hence `Bi`), but where the underlying scalars are the same. Because of that, it has the same module parameters as both `A.M.Definitions.Left` and `A.M.Definitions.Right`, but `Commutative` shouldn't really go in either